Mathlib/Data/Nat/Choose/Basic.lean

```diff
@@ -65,6 +65,9 @@ theorem choose_succ_succ (n k : ℕ) : choose (succ n) (succ k) = choose n k + c
   rfl
 #align nat.choose_succ_succ Nat.choose_succ_succ
 
+theorem choose_succ_succ' (n k : ℕ) : choose (n + 1) (k + 1) = choose n k + choose n (k + 1) :=
+  rfl
```

Comment:
This was added because `rw` was not firing on `choose_succ_succ` when the expression to be rewritten is `n+1` as opposed to `succ n`.